ui/src/views/Student/Student.jsx

```diff
@@ -34,25 +36,27 @@ function Student({ user }) {
 
 
     useEffect(() => {
-        const fetchData = async () => {
+        //     const fetchData = async () => {
```

Comment:
What's the rationale behind commenting this out?

Reply:
Instead of doing a GET request, I can just use the user id passed in. I just forgot to remove the comments! Removing on next push!